### PR TITLE
[MediaBundle]: bulk move media in folders

### PR DESCRIPTION
**Pull request by sandergo90**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1333

This PR adds to possibility to move media in bulk to another folder.

screen shot 2018-03-30 at 15 53 19
screen shot 2018-03-30 at 15 53 26
screen shot 2018-03-30 at 15 53 35

### CODE REVIEW

File: src/Kunstmaan/MediaBundle/Controller/MediaController.php [hunk 1]
```diff
@@ -472,4 +493,49 @@ public function moveMedia(Request $request)
...
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $folder = $form->getData()['folder'];
+            $mediaIds = explode(',', $form->getData()['media']);
+
+            foreach ($mediaIds as $mediaId) {
+                /** @var Media $media */
+                $media = $mediaRepo->getMedia($mediaId);
```
Comment (mlebkowski) [inline src/Kunstmaan/MediaBundle/Controller/MediaController.php]: Fetching from database in a loop? Not a good idea. A simple `UPDATE` query would do fine here.

Comment (sandergo90) [reply src/Kunstmaan/MediaBundle/Controller/MediaController.php]: @mlebkowski Correct! Changed ;)

File: src/Kunstmaan/MediaBundle/Controller/MediaController.php [hunk 2]
```diff
@@ -472,4 +493,46 @@ public function moveMedia(Request $request)
...
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var Folder $folder */
+            $folder = $form->getData()['folder'];
+            $mediaIds = explode(',', $form->getData()['media']);
+
+            foreach ($mediaIds as $mediaId) {
+                $mediaRepo->moveToFolder($mediaId, $folder->getId());
```
Comment (mlebkowski) [inline src/Kunstmaan/MediaBundle/Controller/MediaController.php]: This is still executing queries in a loop. I was thinking more like:

```php
$mediaRepo->createQueryBuilder('media')
    ->update()
    ->set('media.folder', $folder->getId())
    ->where('media.id in (:mediaIds)')
    ->setParameter('mediaIds', $mediaIds)
    ->getQuery()
    ->execute();
```

Comment (sandergo90) [reply src/Kunstmaan/MediaBundle/Controller/MediaController.php]: True, didn't see that ;) I've changed it now.

**Pull request merged**